sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -111,6 +111,14 @@ statement
         (AS? query)?                                                   #createHiveTable
     | CREATE TABLE (IF NOT EXISTS)? target=tableIdentifier
         LIKE source=tableIdentifier locationSpec?                      #createTableLike
+    | replaceTableHeader ('(' colTypeList ')')? tableProvider
+        ((OPTIONS options=tablePropertyList) |
```

Comment:
Should OPTIONS be supported in v2? Right now, we copy options into table properties because v2 has no separate options. I also think it is confusing to users that there are table properties and options.

Reply:
In general I copied this entirely from the equivalent create table statement. How does the syntax for `REPLACE TABLE` differ from that of the existing `CREATE TABLE`? My understanding is `REPLACE TABLE` is exactly equivalent to `CREATE TABLE` with the exception of not having an `IF NOT EXISTS` option.

Reply:
The chunk I copied I believe is https://github.com/apache/spark/pull/24798/files/bc8d3b568988c904dff4f31a19f6dc0aa33f4a8b#diff-8c1cb2af4aa1109e08481dae79052cc3R93.

Reply:
True, it should be the same a `CREATE TABLE`. That's a good reason to carry this forward.